### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by fmedinac**
Add revised Portuguese language translation.

### CODE REVIEW

File: build-serbian.sh [hunk 1]
```diff
@@ -25,3 +25,7 @@ rm ./_site/sr/en.html
 mkdir ./_site/ru
 cp ./sr/ru.html ./_site/ru/index.html
 rm ./_site/sr/ru.html
+
+mkdir ./_site/pt-br
+cp ./sr/pt-br.html ./_site/pt-br/index.html
```
Comment (Stanko) [inline build-serbian.sh:30]: Let's create this file, just copy-paste `en.html`, it should just contains redirection to https://lettersfromsarajevo.com/pt-br/
```
./sr/pt-br.html
```

File: _data/translations/pt-br.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+metaImage: letters-from-sarajevo.jpg # TODO
+
+firstLetterLink: "/pt-br/primeira-carta"
+firstLetterText: "Primeira carta"
+
+prevLinkText: Carta anterior
+nextLinkText: Próxima carta
+
+homeLinkText: Home page
```
Comment (Stanko) [inline _data/translations/pt-br.yml]: Should we translate this, or is this common in Brasil?

File: _data/translations/pt-br.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+title: Cartas de Sarajevo
+
+url: "https://lettersfromsarajevo.com/pt-br/"
+
+metaDescription: >-
+  O começo da Guerra da Bósnia.
+  Kosta, um garoto de dez anos, foge de Sarajevo com sua mãe e irmão mais novo. Seu pai fica na cidade, sitiada e bombardeada, sobrevivendo a tempos difíceis com outros sarajevanos. Apesar de tudo, ele escreve cartas divertidas para seus filhos, tentando encorajá-los e animá-los.
+  Dez anos depois, usando as cartas originais escritas por seu pai, Kosta escreve a história de sua infância.
+
+metaImage: letters-from-sarajevo.jpg # TODO
```
Comment (Stanko) [inline _data/translations/pt-br.yml:12]: Let's keep this one for now, I'll create new one at some point.

**Review by Stanko — APPROVED** [review]
❤️ for the 100th time - thank you for doing this 🙂 

This is great, please create that redirection file for pt-br and it will be ready to be deployed!

**Comment by Stanko** [comment]
Btw preview is here:
https://5e8273fbc5be7b0007ace56c--lettersfromsarajevo.netlify.com/pt-br/

File: _data/translations/pt-br.yml (hunk 2 above)
Comment (fmedinac) [reply _data/translations/pt-br.yml]: We often use "Home page" here in Brasil, it could be "Página inicial" as well, both work.

**Pull request merged**